app/api/views/briefs.py

```diff
@@ -422,6 +433,20 @@ def update_brief(brief_id):
             for seller_code, seller in brief.data['sellers'].iteritems():
                 supplier = suppliers.get_supplier_by_code(seller_code)
                 send_seller_invited_to_rfx_email(brief, supplier)
+        try:
+            brief_url_external = '{}/2/digital-marketplace/opportunities/{}'.format(
+                current_app.config['FRONTEND_ADDRESS'],
+                brief_id
+            )
+            _notify_team_brief_published(
```

Comment:
By being after the `brief.publish()` call and the seller email sending, this call should not interfere with the actual publishing action should it fail or malfunction. I tested in dev and an unreachable Slack webhook endpoint results in a logged error but an undisrupted experience for the publishing user.